src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/BOM/MaterialForm/MaterialFormView.tsx

```diff
@@ -89,9 +91,15 @@ const MaterialFormView: React.FC<MaterialFormViewProps> = ({
               defaultValue={control._defaultValues.status}
               render={({ field }) => (
                 <TextField {...field} select variant="outlined" error={!!errors.status} helperText={errors.status?.message}>
-                  {['ORDERED', 'UNORDERED', 'SHIPPED', 'RECEIVED'].map((status) => (
+                  {[
+                    MaterialStatus.Ordered,
+                    MaterialStatus.Received,
+                    MaterialStatus.Shipped,
+                    MaterialStatus.NotReadyToOrder,
+                    MaterialStatus.ReadyToOrder
+                  ].map((status) => (
                     <MenuItem key={status} value={status}>
-                      {status}
+                      {displayEnum(status)}
```

Comment:
getting this rn, so you might need to add `status ? displayEnum(status) : ''`, so this doesn't happen:
![image](https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/00f68ae8-11b9-4320-a553-a5e52c885008)

Reply:
Just changed it